setup.cfg

```diff
@@ -1,7 +1,7 @@
 [flake8]
 max-line-length = 120
 exclude = .tox,.git,*/migrations/*,*/static/CACHE/*,docs,node_modules,venv
-ignore = I001, I002, I003, I004
```

Comment:
Can you update the commit message to indicate what you're excluding here

Reply:
Done: mostly I was irritated at how isort gave no useful messages for diagnosing what was missing!